lib/go/templates/flow_arcade_token_templates.go

```diff
@@ -0,0 +1,81 @@
+package templates
+
+import (
+	"strings"
+
+	"github.com/onflow/flow-core-contracts/lib/go/templates/internal/assets"
+)
+
+const (
+	addMinterFilename             = "flowArcadeToken/admin/add_minter.cdc"
+	deployFlowArcadeTokenFilename = "flowArcadeToken/admin/deploy_flow_arcade_token.cdc"
+	mintTokensFilename            = "flowArcadeToken/mint_tokens.cdc"
+	setupAccountFilename          = "flowArcadeToken/setup_account.cdc"
+	transferTokensFilename        = "flowArcadeToken/transfer_tokens.cdc"
+
+	getBalanceFilename = "flowArcadeToken/scripts/get_balance.cdc"
+	getSupplyFilename  = "flowArcadeToken/scripts/get_supply.cdc"
+
+	defaultFungibleTokenAddress   = "FUNGIBLETOKENADDRESS"
+	defaultFlowArcadeTokenAddress = "ARCADETOKENADDRESS"
+)
+
+// ReplaceAddresses replaces the contract addresses in the code
+func replaceAddresses(code, ftAddr, fatAddr string) string {
```

Comment:
```suggestion
func replaceFATAddress(code, fatAddr string) string {
```

we already have the `replaceAddresses` function that replaces fungible token and flow token addresses, so we should probably rename this and make it only replace the FAT address

Reply:
And change all the templates in the file to call both replace functions. Do you think that sounds ok?

Reply:
Yes that makes sense. I'll make that change